### ISSUE TITLE
In training loss

### ISSUE THREAD

**Issue by akshay-antony** [xinge008/Cylinder3D#141]
Is there a need to convert sparse to dense for loss calculation and backward pass, or can we use the inverse map and calculate loss between two sparse tensors i.e find argmax without using logits.dense() and compare with pointwise labels. Will this work too?
